### PR TITLE
Update doc for cid025

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 1]
```diff
@@ -104,26 +104,26 @@ QuickSight is the AWS Business Intelligence tool that will allow you to not only
 
 2. Click the **Launch CloudFormation button** below to open the **pre-populated stack template** in your CloudFormation console and select **Next**
 
-	- [Launch CloudFormation Template](https://console.aws.amazon.com/cloudformation/home#/stacks/new?&templateURL=https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/cid-cfn.yml&stackName=Cloud-Intelligence-Dashboards)
+	- [Launch CloudFormation Template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?&templateURL=https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/cid-cfn.yml&stackName=Cloud-Intelligence-Dashboards&param_DeployCUDOSDashboard=yes&param_DeployKPIDashboard=yes&param_DeployCostIntelligenceDashboard=yes)
 	
-3. Enter a **Stack name** for your template such as **CID**
-4. Review **Readme Sections** parameter to confirm prerequisites before specifying the other parameters. You must answer 'yes' to all.
+3. Enter a **Stack name** for your template such as **Cloud-Intelligence-Dashboards**
+4. Review **Common Parameters** parameter to confirm prerequisites before specifying the other parameters. You must answer 'yes' to all.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:110]: there is also QuickSight username. I'd suggest to re-phrase 'You must answer 'yes' to all.' to something like: 'You need to confirm prerequisites by answering 'yes' and specify your QuickSight username

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 2]
```diff
@@ -104,26 +104,26 @@ QuickSight is the AWS Business Intelligence tool that will allow you to not only
...
-1. Update your **CURBucketPath** and set the **CURisAggregated** flag to `yes`;
-   
-2. Select the Dashboards you want to install. We recommend deploying all three CUR-based dashboards; Cost Intelligence Dashboard, CUDOS, and the KPI Dashboard.
+1. Update your **CURBucketPath** if needed. 
+   1. (Default) If you used the CFN Template in the CUR setup process above then **CURBucketPath** needs to be the s3 path to the folder in your CUR bucket where account IDs are. For example `s3://cid-1234567890123-shared/cur/`
+   2. If you did *not* use the CFN automated CUR setup process above and have just one CUR you setup manually then **CURBucketPath** needs to be the s3 path to the folder in your CUR bucket where the year folders are. For example `s3://cid-1234567890123-shared/prefix/name/name/` (double check this path, you must see /year=xxxx partitions in there)
+
 
-3. Select **Next** at the bottom of **Specify stack details** and then select **Next** again on the **Configure stack options** page.
+2. Select the Dashboards you want to install. We recommend deploying all three.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:122]: Why not to list them here? We recommend to deploy all three CUR based dashboards - Cost Intelligence Dashboard, CUDOS, and the KPI Dashboard

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 1 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:110]: QuickSight username is just bellow

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 2 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:122]: Just think we need to keep this as simple as possible. I do not see additional value in adding names here. Why not keeping this as is?

Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:122]: Also it is not related to this change so resolving.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 1 above)
Comment (yprikhodko) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:110]: QuickSight username is a part of common parameters. You must answer 'yes' to all might confuse.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 2 above)
Comment (yprikhodko) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:122]: It's related to this change as we list dashboard names currently (line 120) and you removed them here. Please use the same phrase as is now 'We recommend deploying all three CUR-based dashboards; Cost Intelligence Dashboard, CUDOS and the KPI Dashboard.'

Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:122]: Yes indeed.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 1 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md]: I propose other option. Please feel free to propose your option for this paragraph.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 2 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md]: Put it back.

**Review by yprikhodko — APPROVED** [review]
Thanks for adjusting the text

**Pull request merged**